api-samples/webRequest/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "webRequest Demo",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
+  },
+  "permissions": [
+    "webRequest",
+    "webRequestAuthProvider",
+    "<all_urls>",
```

Comment:
> "<all_urls>"

This should go under host permissions. @oliverdunk Do we really need access to all sites for this example?

Reply:
I think we should be fine just requesting whatever site we use for the demo :)